src/command_modules/azure-cli-role/azure/cli/command_modules/role/commands.py

```diff
@@ -84,6 +84,7 @@ def load_command_table(self, _):
         g.custom_command('delete', 'delete_application')
         g.custom_command('list', 'list_apps')
         g.custom_show_command('show', 'show_application')
+        g.custom_command('grant', 'grant_application')
```

Comment:
Does AAD graph support "permission revoke"? If yes, I suggest you author it like `az ad app permission grant` so we can add `revoke` and `list` later

Reply:
Changed to ```g.custom_command('permission grant', 'grant_application')```